src/EFCore.PG/Scaffolding/Internal/NpgsqlDatabaseModelFactory.cs

```diff
@@ -508,6 +539,14 @@ ELSE pg_get_expr(indpred, cls.oid)
                         if (record.GetValueOrDefault<string>("amname") is string indexMethod && indexMethod != "btree")
                             index[NpgsqlAnnotationNames.IndexMethod] = indexMethod;
 
+                        // Handle index operator classes, which we pre-loaded
+                        var opClassNames = record.GetValueOrDefault<uint[]>("indclass")
+                            .Select(oid => opClasses.TryGetValue(oid, out var opc) && !opc.IsDefault
+                                ? opc.Name
+                                : null).ToArray();
```

Comment:
nit: This unaligned LINQ wrapping is hard to read...

The current version appears to end at column 110, while something like this ends around 114:

```c#
var opClassNames =
    record.GetValueOrDefault<uint[]>("indclass")
          .Select(oid => opClasses.TryGetValue(oid, out var opc) && !opc.IsDefault ? opc.Name : null)
          .ToArray();
```